agent/config/config.go

```diff
@@ -34,7 +35,7 @@ func FormatFrom(name string) string {
 }
 
 // Parse parses a config fragment in either JSON or HCL format.
-func Parse(data string, format string) (c Config, md mapstructure.Metadata, err error) {
+func Parse(data string, format string) (c Config, keys []string, err error) {
```

Comment:
Didn't you suggest in my aclv8 PR to do the reverse of this?

Reply:
Must have been an issue with rebasing. I didn't mean to touch this func.